PowerPoint Unzipped File Experiments/OneImageAndPlainText - Readable XML Formatting.pptx/docProps/core.xml

```diff
@@ -1,9 +1,9 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<?xml version="1.0" encoding="UTF-8"?>
 <cp:coreProperties xmlns:cp="http://schemas.openxmlformats.org/package/2006/metadata/core-properties" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:dcmitype="http://purl.org/dc/dcmitype/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <dc:title>Slide 1</dc:title>
-  <dc:creator>node-pptx</dc:creator>
-  <cp:lastModifiedBy>node-pptx</cp:lastModifiedBy>
-  <cp:revision>1</cp:revision>
-  <dcterms:created xsi:type="dcterms:W3CDTF">2017-12-06T03:55:22Z</dcterms:created>
-  <dcterms:modified xsi:type="dcterms:W3CDTF">2017-12-06T03:55:50Z</dcterms:modified>
+  <dc:creator>gregdolley</dc:creator>
```

Comment:
i'd rather  this say node-pptx so that it's clear that it's the base template

Reply:
Same as above - it's in that experimental test folder. Probably PowerPoint added this.